pkg/solana/codec/testutils/itemArray1TypeIDL.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
I'm curious if those idls are purely mock data or do we have corresponding program somewhere?

Reply:
Based on where they're used and the names inside of them, it looks to be just for tests

Reply:
yeah, just wrote them "manually" no corresponding program